roxie/udplib/uttest.cpp

```diff
@@ -1120,7 +1120,7 @@ int main(int argc, char * argv[] )
     if (modeType & RCV_MODE_BIT) 
     {
         rcvMgr = createReceiveManager(7000, 7001, 7002, 7003, multiCast, 100, 0x7fffffff, myIndex);
-        rowMgr = createRowManager(0, NULL, queryDummyContextLogger(), NULL);
+        rowMgr = createRowManager(0, NULL, queryDummyContextLogger(), NULL, false, false);
```

Comment:
this looks like it has 1 parameter too many

Reply:
Hmmm - compiled on my machine.

Reply:
Ok, fixed (this is in commented-out code). The compile error arose from the rebase, also fixed